src/main/webapp/app/common/domain/User.ts

```diff
@@ -0,0 +1,10 @@
+export class UserCredentialsDTO {
```

Comment:
Extract this class from this MR and create another one specificly for User management